library/src/c_api.rs

```diff
@@ -296,13 +299,15 @@ mod test {
         )
     }
 
+    #[serial]
```

Comment:
Minor but it would be nice to create a module for tests as is described in the docs:
https://doc.rust-lang.org/book/ch11-03-test-organization.html

```rs
#[cfg(test)]
mod tests {
    #[test]
    fn it_works() {
        let result = 2 + 2;
        assert_eq!(result, 4);
    }
}
```

Reply:
I think we do that already? Or am I misunderstanding.  As it notes in the diff, this is within "mod test"